Utils/interface/ParticleMapper.h

```diff
@@ -0,0 +1,41 @@
+//--------------------------------------------------------------------------------------------------
+// Particle Mapper
+//
```

Comment:
Needs much more substantial documentation to explain what it is. This is the only place where we write things about modules. In simple classes this level is fine, here we need some more description.